docs/pages/reference/config.mdx

```diff
@@ -3,64 +3,85 @@ title: Teleport Configuration Reference
-<Tabs>
-<TabItem scope={["oss", "enterprise"]} label="Self-Hosted">
+## Before using this reference
 
-- **Do not use this example configuration in production.**
+<Notice type="danger">
 
-  You must edit your configuration file to meet the needs of your environment.
-  Using a copy of the reference configuration will have unintended effects. To
-  create a configuration file that you can use as a starting point, run the
-  following command:
+Do not use this example configuration in production.
 
-  ```code
-  $ teleport configure -o file
-  ```
+</Notice>
 
-- You should back up your configuration file before making changes. This will
-  enable you to roll back to the previous configuration if you need to.
+You must edit your configuration file to meet the needs of your environment.
+Using a copy of the reference configuration will have unintended effects. To
+create a configuration file that you can use as a starting point, run the
+following command:
 
-</TabItem>
-<TabItem scope={["oss", "enterprise"]} label="Teleport Cloud">
+```code
+$ teleport configure -o file
+```
+
+There are also `configure` commands available for the SSH Service and Database
+Service. See our documentation on `teleport node configure` and `teleport db
+configure` in the [Teleport CLI Reference](cli.mdx#teleport).
+
+<Notice type="warning">
+
+You should back up your configuration file before making changes. This will
+enable you to roll back to the previous configuration if you need to.  
+
+</Notice>
+
+## Enabling Teleport services
+
+The `teleport` process can run multiple services. 
 
-- **Do not use this example configuration in production.**
+For some services, you must enable the service within your Teleport
+configuration in order to start it. Other services are enabled by default. 
 
-  You must edit your configuration file to meet the needs of your environment.
-  Using a copy of the reference configuration will have unintended effects. To
-  create a configuration file that you can use as a starting point, run the
-  following command:
+To enable or disable a service, include the following in your Teleport
+configuration, replacing `service_name` with the name of your service (service
+names are listed below):
 
-  ```code
-  $ teleport configure -o file
-  ```
+```yaml
+service_name:
+  enabled: "no"
+```
 
-- You should back up your configuration file before making changes. This will
-  enable you to roll back to the previous configuration if you need to.
+Teleport supports the following services:
 
-- Teleport Cloud manages the Auth Service and Proxy Service for you. Your
-  Teleport resource services (e.g., the Application Service and Database
-  Service) should include the following configuration options to avoid
-  unintended effects:
+|Service|Configuration section|Enabled by default|
+|---|---|---|
+|Application Service|`app_service`|❌|
+|Auth Service|`auth_service`|✅|
+|Database Service|`db_service`|❌|
+|Discovery Service|`discovery_service`|❌|
```

Comment:
Does `discovery_service` exist in v10? @r0mant @lxea

Reply:
Yes.